src/nacl/bindings/crypto_pwhash_scryptsalsa208sha256.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2013 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function
+
+from nacl._sodium import ffi, lib
+from nacl.exceptions import CryptoError
+
+__all__ = [
+    "crypto_pwhash_scryptsalsa208sha256",
+    "crypto_pwhash_scryptsalsa208sha256_str",
+    "crypto_pwhash_scryptsalsa208sha256_str_verify",
+    "SALTBYTES",
+    "STRBYTES",
+    "OPSLIMIT_INTERACTIVE",
+    "MEMLIMIT_INTERACTIVE",
+    "OPSLIMIT_SENSITIVE",
+    "MEMLIMIT_SENSITIVE"
+]
+
+
+SALTBYTES = lib.crypto_pwhash_scryptsalsa208sha256_saltbytes()
+STRBYTES = lib.crypto_pwhash_scryptsalsa208sha256_strbytes()
+OPSLIMIT_INTERACTIVE = \
+    lib.crypto_pwhash_scryptsalsa208sha256_opslimit_interactive()
+MEMLIMIT_INTERACTIVE = \
+    lib.crypto_pwhash_scryptsalsa208sha256_memlimit_interactive()
+OPSLIMIT_SENSITIVE = \
+    lib.crypto_pwhash_scryptsalsa208sha256_opslimit_sensitive()
+MEMLIMIT_SENSITIVE = \
+    lib.crypto_pwhash_scryptsalsa208sha256_memlimit_sensitive()
+
+
+def crypto_pwhash_scryptsalsa208sha256(outlen, passwd, salt,
+                                       opslimit=OPSLIMIT_SENSITIVE,
+                                       memlimit=MEMLIMIT_SENSITIVE
+                                       ):
+    """
```

Comment:
I was forgetting: another reason why I've defined the short local names was trying to get along with flake8 both in the function definitions and in the docstrings.